src/app/products/product.model.ts

```diff
@@ -0,0 +1,12 @@
+export class Product {
```

Comment:
thought we were letting wesley do this

Reply:
Just a template. I Already told to him change it to our model